database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -18,6 +18,7 @@ public function up(): void
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
             $table->rememberToken();
+            $table->foreignId('role_id')->nullable();
```

Comment:
Please add this column in a separate migration. 

The best practice and the reason for this is that when you share the code with other developers or the project gets deployed to staging or production, there will be no way to add this column since the migration was already executed on that environment automatically.

Reply:
Done.